ControlExtention/src/test/java/Slider.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Slider extends PageObject{
+	private WebElement slider;
+
+	public Slider(WebDriver driver, String baseURL) {
```

Comment:
Control Extensions are wrappers for WebElements. There should be no reference to urls or pages inside the Control Extension.